sql/core/src/test/resources/sql-tests/inputs/charvarchar.sql

```diff
@@ -61,7 +61,60 @@ desc formatted char_part;
 MSCK REPAIR TABLE char_part;
 desc formatted char_part;
 
+create table char_tbl4(c char(5), v varchar(6)) using parquet;
```

Comment:
We don't need tests for the mixed case of string types, too?

Reply:
yes, there are a lot of strange cases too :)